ingest-app/src/cmr/ingest/api/subscriptions.clj

```diff
@@ -48,7 +48,18 @@
 (defn- check-subscription-ingest-permission
   "All the checks needed before starting to process an ingest of subscriptions"
   [concept request-context headers provider-id]
-  (let [subscription-user (:SubscriberId (json/decode (:metadata concept) true))
+  (let [concept-id (:concept-id concept)
+        subscription-user (if concept-id
+                            (-> (mdb/find-concepts request-context
+                                                   {:provider-id provider-id
+                                                    :native-id (:native-id concept)
+                                                    :exclude-metadata false
+                                                    :latest true}
+                                                   :subscription)
+                                first
+                                :extra-fields
+                                :subscriber-id)
+                            (:SubscriberId (json/decode (:metadata concept) true)))
```

Comment:
Let me try to understand this: when concept-id exists, it's an update, otherwise it's a create.
When try to update, the token user needs to be the same as the subscriber-id of the existing subscription fetched, rather than the subscriber-id in the metadata because it could be different from the original subscriber, that looks good.  When ingesting a brand new subscription, any registered user can create a subscription as long as the subscriber-id is the same as the token-user. Correct?